### PR TITLE
[Bug Template] New format bug template

### PR DESCRIPTION
**Pull request by hbalasu1**
This new bug template provides an easy interface for customers to fill in the information before submitting a new issue.

### Details:
 - Adding new bug template bug.yml
 - 

### Tickets:
 - NIL

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    attributes:
+      value: |
+        Please provide all the necessary information to expedite the response.
+        #Start here
+  - type: input
+    id: ov_version
+    attributes:
+      label: OpenVINO Version 
+      description: OpenVINO Version/Branch/tag
+      placeholder: 2022.3.0 / Master Branch / tag 2022.3.0
```
Comment (ilya-lavrenov) [inline .github/ISSUE_TEMPLATE/bug.yml:18]: some people even use older LTS releases like 2021.4

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        - macOS 11
+        - macOS 12
+        - Other (Please specify in description)
+    validations:
+      required: true
+  - type: dropdown
+    id: device_use
+    attributes:
+      label: Device Use for Inference Engine
+      description: Device Use for Inference Engine
```
Comment (ilya-lavrenov) [inline .github/ISSUE_TEMPLATE/bug.yml:42]: let's drop engine (it refers to old OV name "Inference Engine")

**Review by ilya-lavrenov — COMMENTED** [review]
LGTM in general!
Do you think we also need to create more template pages like questions?

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+  - type: dropdown
+    id: device_use
+    attributes:
+      label: Device Use for Inferencing
+      description: Device Use for Inferencing
+      options:
+        - CPU
+        - GPU
+        - GNA
+        - NCS2 (intel Movidius)
```
Comment (p-wysocki) [inline .github/ISSUE_TEMPLATE/bug.yml:48]:
```suggestion
        - NCS2 (Intel Movidius)
```

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+      label: Framework
+      description: Framework use in modal optimization
+      options:
+        - TensorFlow 1
+        - Keras (TensorFlow 2)
+        - Caffe
+        - ONNX
+        - Pytorch
+        - mxnet
+        - Paddle Paddle
```
Comment (p-wysocki) [inline .github/ISSUE_TEMPLATE/bug.yml:67]:
```suggestion
        - PaddlePaddle
```

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+    id: framework
+    attributes:
+      label: Framework
+      description: Framework use in modal optimization
+      options:
+        - TensorFlow 1
+        - Keras (TensorFlow 2)
+        - Caffe
+        - ONNX
+        - Pytorch
```
Comment (p-wysocki) [inline .github/ISSUE_TEMPLATE/bug.yml:65]:
```suggestion
        - PyTorch
```

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+        - mxnet
+        - Paddle Paddle
+    validations:
+      required: true
+  - type: input
+    id: model_name
+    attributes:
+      label: Model Name  
+      description: Please provide us the link to your model in the description
```
Comment (p-wysocki) [inline .github/ISSUE_TEMPLATE/bug.yml:74]: Do we have any preferred web storage? It's often a risk to download and open "random" .zip files and so on.

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+      placeholder: ResNet50 / YOLOv4
+    validations:
+      required: false
+      #Endhere 
+  - type: textarea
+    id: bug_description
+    attributes:
+      label: Details Description of the issue
+      description: Also, tell us the issue, and what did you expect to happen?
+      placeholder: Please provide a details description of what happened
```
Comment (p-wysocki) [inline .github/ISSUE_TEMPLATE/bug.yml:84]:
```suggestion
      placeholder: Please provide a detailed description of what happened
```

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 8]
```diff
@@ -0,0 +1,113 @@
...
+      label: Model Name  
+      description: Please provide us the link to your model in the description
+      placeholder: ResNet50 / YOLOv4
+    validations:
+      required: false
+      #Endhere 
+  - type: textarea
+    id: bug_description
+    attributes:
+      label: Details Description of the issue
```
Comment (p-wysocki) [inline .github/ISSUE_TEMPLATE/bug.yml:82]:
```suggestion
      label: Issue description
```

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 9]
```diff
@@ -0,0 +1,113 @@
...
+      description: Also, tell us the issue, and what did you expect to happen?
```
Comment (p-wysocki) [inline .github/ISSUE_TEMPLATE/bug.yml:83]:
```suggestion
      description: What issue are you having and what did you expect to happen instead?
```

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 10]
```diff
@@ -0,0 +1,113 @@
...
+      placeholder: Please provide a details description of what happened
+      value: "Error when performing Modal optimization on yolov4 model."
+    validations:
+      required: true
+  - type: textarea
+    id: step_by_step
+    attributes:
+      label: Details Step by Step 
+      description: Details instructions on reproducing the issue
+      placeholder: Please provide us Step by step on how we can reproduce the issue
```
Comment (p-wysocki) [inline .github/ISSUE_TEMPLATE/bug.yml:93]:
```suggestion
      label: Step by step reproduction 
      description: How can we reproduce your issue?
      placeholder: Please provide a detailed instruction on how to reproduce the issue
```

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 11]
```diff
@@ -0,0 +1,113 @@
...
+  - type: markdown
+    attributes:
+      value: |
+        Please provide all the necessary information to expedite the response.
+        #Start here
+  - type: input
+    id: ov_version
+    attributes:
+      label: OpenVINO Version 
+      description: OpenVINO Version or Branch/tag in OpenVINO Github
```
Comment (p-wysocki) [inline .github/ISSUE_TEMPLATE/bug.yml:17]:
```suggestion
      description: OpenVINO version, branch or tag in OpenVINO GitHub
```

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 12]
```diff
@@ -0,0 +1,113 @@
...
+      placeholder: 2021.4.0 LTS / Master Branch / tag 2022.3.0
+    validations:
+      required: false
+  - type: dropdown
+    id: os
+    attributes:
+      label: Operating System
+      description: Operating System
```
Comment (p-wysocki) [inline .github/ISSUE_TEMPLATE/bug.yml:25]:
```suggestion
      description: What OS are you using?
```

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 13]
```diff
@@ -0,0 +1,113 @@
...
+        - ONNX
+        - Pytorch
+        - mxnet
+        - Paddle Paddle
+    validations:
+      required: true
+  - type: input
+    id: model_name
+    attributes:
+      label: Model Name
```
Comment (p-wysocki) [inline .github/ISSUE_TEMPLATE/bug.yml:73]:
```suggestion
      label: Model used
```

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 14]
```diff
@@ -0,0 +1,113 @@
...
+      required: true
+  - type: input
+    id: model_name
+    attributes:
+      label: Model Name  
+      description: Please provide us the link to your model in the description
+      placeholder: ResNet50 / YOLOv4
+    validations:
+      required: false
+      #Endhere
```
Comment (p-wysocki) [inline .github/ISSUE_TEMPLATE/bug.yml:78]: Are these comments supposed to be here?

File: .github/ISSUE_TEMPLATE/bug.yml (hunk 1 above)
Comment (p-wysocki) [reply .github/ISSUE_TEMPLATE/bug.yml]: I think it's just a placeholder, a text which disappears once you click the text field. I guess we can add "Example: ..." before that to further indicate that it's just a suggestion, and not a list of available options.

**Review by hbalasu1 — COMMENTED** [review]
Done amend and submit for review

**Review by ilya-lavrenov — APPROVED** [review]

**Review by p-wysocki — APPROVED** [review]

**Review by ilyachur — APPROVED** [review]

**Review by p-wysocki — COMMENTED** [review]
Shouldn't https://github.com/openvinotoolkit/openvino/blob/master/.github/ISSUE_TEMPLATE/bug.md be deleted in this PR? I think this template should replace the old `.md` file, if this PR gets merged we have old `bug.md` and new `bug.yml` in the same directory. Is this intended? How will GitHub behave?

**Pull request merged**